deployer.py

```diff
@@ -253,13 +261,13 @@ def set_scheduler(self, version):
 
     def deploy(self, environment='pre'):
         self.create_application_if_not_present()
-        elbs = []
+        self.elbs = []
```

Comment:
why do you want the `elbs` var as an instance var if you're not using it anywhere else